packages/fuselage-toastbar/src/ToastBar.spec.tsx

```diff
@@ -1,12 +1,63 @@
 import { composeStories } from '@storybook/testing-react';
-import { render } from '@testing-library/react';
+import { render, getByRole, screen, fireEvent } from '@testing-library/react';
 
-import * as stories from './stories';
+import * as stories from './ToastBar.stories';
+import ToastBarProvider from './ToastBarProvider';
 
-const { ToastBarWithData } = composeStories(stories);
+const { Default, TopEnd } = composeStories(stories, {
+  decorators: [
+    (Story) => (
+      <ToastBarProvider>
+        <Story />
+      </ToastBarProvider>
+    ),
+  ],
+});
+
+const topEndStyle = {
+  top: '0',
+  right: '0',
+};
+
+const topStartStyle = {
+  top: '0',
+  left: '0',
+};
+
+describe('[fuselage-toastbar]', () => {
```

Comment:
No need for a top-level `describe` in the test suite.

Reply:
I do prefer to use describe to split below the render tests and behavior tests. Kinda pattern we use on fuselage lib

Reply:
fixed!